cdk/lib/recipes-backend.ts

```diff
@@ -335,6 +340,45 @@ export class RecipesBackend extends GuStack {
 			{ suffix: 'curation.json' },
 		);
 
+		new GuScheduledLambda(this, 'PublishContributors', {
+			app: 'recipes-publish-contributor-information',
+			architecture: Architecture.ARM_64,
+			fileName: 'profile-cache-rebuild.zip',
+			functionName: `PublishRecipeContributors-${this.stage}`,
+			handler: 'main.handler',
+			initialPolicy: [
+				new PolicyStatement({
+					effect: Effect.DENY,
+					actions: ['*'],
+					resources: [serving.staticBucket.bucketArn + '/content/*'],
+				}),
+				new PolicyStatement({
+					effect: Effect.ALLOW,
```

Comment:
I think we don't need to provide ALLOW property, recently I came across with the situation to confirm where I was able to get S3 work done without this.  
it is auto default to ALLOW i think, so just fyi and not to change here anything.